jt400-override-ccsid/readme.md

```diff
@@ -0,0 +1,30 @@
+
+# Purpose
+
+Override the ccsid and force the character encoding
+
+really you should not need this and just set the right encoding on the tables sadly if you have a large legacy code base and a large number of customers who have this incorrectly configured that correct solution might be costly and risky
```

Comment:
It's not so much having the wrong CCSID on the tables that is the problem. The system can transparently handle conversions between the encodings and make this work, provided the characters are present in both code pages.

The issue is specifically with a [system value called QCCSID](https://www.ibm.com/docs/en/i/7.5?topic=faqs-i-system-value-qccsid). If this is set to 65535 (binary), it indicates that jobs will have no CCSID, and no conversion will be performed.

Reply:
Actually, here's a slightly better description of the problem

https://www.ibm.com/support/pages/what-impact-changing-qccsid-shipped-65535-another-ccsid